test/unit/test_mail_quoting.js

```diff
@@ -393,8 +415,9 @@ commonCase('Quoting', function(T) {
         eCheck.expect(tdef.chunks[i], tdef.chunks[i+1]);
       }
       var roundtrip = true;
-      if (tdef.hasOwnProperty('roundtrip'))
+      if ('roundtrip' in tdef) {
```

Comment:
I have no clue why I did this.  This part of the patch dates back ~5 months as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1169130.  My guess would be that I was surprised by the forward-roundtripping functionality while debugging and did not realize that it was already an override gated on property presence and started blindly poking things in confusion before I realized what was going on.  And then I probably failed to revert this.

The test definitely passes either way; I'm fine to change it back.